### PR TITLE
Katello 3.3 docs

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@jlsherrill 3.3 docs for your review

**Comment by jlsherrill** [comment]
is something missing here?

**Comment by ehelms** [comment]
@jlsherrill A whole lot of somethings. Updated.

**Comment by jlsherrill** [comment]
* Clicking on the 3.3-RC link on this page: /plugins/katello/  takes me to /plugins/katello/3.3-RC/index.html and throws a 404
* all the left hand links on this:  /plugins/katello/3.3/index.html take you to the nightly versions of those pages instead of the 3.3 versions
* On this page: /plugins/katello/3.3/installation/index.html  it states `These instructions are for installing Katello 3.3, but the latest stable is .`
* 3.3 install instructions on  /plugins/katello/3.3/installation/index.html  say to use: ` yum -y localinstall http://yum.theforeman.org/releases/3.3/el7/x86_64/foreman-release.rpm`  but should be 1.14  (same for upgrade instructions)

**Comment by ehelms** [comment]
Updated to address all comments.

File: plugins/katello/3.3/release_notes/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% else %}
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+{% endif %}
+
+## Features
+
+## Deprecation Warnings
+
+## Contributors
+
```
Comment (jlsherrill) [inline plugins/katello/3.3/release_notes/release_notes.md:20]: did you want to go ahead and generate and add the release notes?

File: plugins/katello/3.3/cli/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+layout: plugins/katello/documentation
+title: CLI
+version: 3.2
```
Comment (jlsherrill) [inline plugins/katello/3.3/cli/index.md]: should be 3.3

File: plugins/katello/3.3/release_notes/release_notes.md (hunk 1 above)
Comment (ehelms) [reply plugins/katello/3.3/release_notes/release_notes.md:20]: I think that might be better as a separate commit (easier to review and comment on).

**Comment by ehelms** [comment]
Updated

**Comment by jlsherrill** [comment]
ACK

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**